mathesar_ui/package.json

```diff
@@ -7,76 +7,69 @@
     "test": "jest",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook"
   },
-  "resolutions": {
-    "trim": "1.0.1",
-    "browserslist": "4.16.7",
-    "glob-parent": "6.0.1",
-    "tar": "6.1.11"
-  },
   "devDependencies": {
-    "@babel/core": "^7.14.6",
-    "@babel/preset-env": "^7.14.7",
-    "@storybook/addon-actions": "^6.3.6",
-    "@storybook/addon-essentials": "^6.3.6",
-    "@storybook/addon-links": "^6.3.6",
-    "@storybook/addon-svelte-csf": "^1.1.0",
-    "@storybook/builder-webpack5": "^6.3.7",
-    "@storybook/manager-webpack5": "^6.3.7",
-    "@storybook/svelte": "^6.3.6",
-    "@sveltejs/vite-plugin-svelte": "^1.0.0-next.21",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/svelte": "^3.0.3",
-    "@testing-library/user-event": "^14.0.0-beta.13",
-    "@tsconfig/svelte": "^1.0.13",
-    "@types/jest": "^26.0.23",
-    "@types/js-cookie": "^2.2.6",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
-    "@vitejs/plugin-legacy": "^1.5.2",
-    "babel-jest": "^26.6.3",
-    "babel-loader": "^8.2.2",
-    "css-loader": "^6.5.1",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb-typescript": "^12.3.1",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-import-resolver-typescript": "^2.5.0",
+    "@storybook/addon-actions": "^6.5.12",
+    "@storybook/addon-essentials": "^6.5.12",
+    "@storybook/addon-links": "^6.5.12",
+    "@storybook/addon-svelte-csf": "^2.0.7",
+    "@storybook/builder-vite": "^0.2.2",
+    "@storybook/client-api": "^6.5.12",
+    "@storybook/svelte": "^6.5.12",
+    "@sveltejs/vite-plugin-svelte": "^1.0.8",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/svelte": "^3.2.1",
+    "@testing-library/user-event": "^14.4.3",
+    "@tsconfig/svelte": "^3.0.0",
+    "@types/jest": "^29.0.3",
+    "@types/js-cookie": "^3.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
+    "@typescript-eslint/parser": "^5.38.0",
+    "@vitejs/plugin-legacy": "^2.2.0",
+    "eslint": "^8.23.1",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-import-resolver-typescript": "^3.5.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^24.7.0",
-    "eslint-plugin-promise": "^5.2.0",
-    "eslint-plugin-svelte3": "^3.4.0",
-    "jest": "^27.5.1",
-    "prettier": "^2.5.0",
-    "prettier-plugin-svelte": "^2.5.0",
-    "sass": "^1.35.1",
-    "sass-loader": "^12.4.0",
-    "style-loader": "^3.3.1",
-    "svelte": "^3.42.4",
-    "svelte-check": "^2.2.5",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.0.4",
+    "eslint-plugin-promise": "^6.0.1",
+    "eslint-plugin-svelte3": "^4.0.0",
+    "jest": "^29.0.3",
+    "jest-environment-jsdom": "^29.0.3",
+    "npm": "^8.19.2",
+    "postcss": "^8.4.16",
```

Comment:
Neat! Did you have anything in mind for postcss?

Reply:
Postcss is used internally by svelte-preprocess to make the `:global` rules work. They have removed it from being installed by default in the recent version on svelte-preprocess, so we have to install it manually.

I could see us using more of postcss features, but I do not have anything in mind for it at the moment.